CONTRIBUTING.md

```diff
@@ -39,3 +39,72 @@ Apply Linter checks with the following instructions
 ## Contributing to Issue Finder
 
 - We welcome all meaningful contributions to Issue Finder. If you see any areas of improvement or wish to contribute to existing issues, please connect with our maintainers directly on the issue, and we will be pleased to help you as much as possible.
+
+
+## Contributing - Open Source
+
+**1.** Fork the repository.
+
+**2.** Clone your forked copy of the project into your local system.
+
+```
+git clone --depth 1 https://github.com/yourusername/issue-finder.git
+```
+
+**3.** Navigate to the project directory. 📁
+
+```
+cd healthphilics
+```
+
+**4.** Add a reference (remote) to the original repository.
+
+```
+git remote add upstream https://github.com/voscarmv/issue-finder.git
+```
+
+**5.** Check the remotes for this repository.
+
+```
+git remote -v
+```
+
+**6.** Always take a pull from the upstream repository to your master branch to keep it at par with the main project (updated repository).
+
+```
+git pull upstream main
+```
+
+**7.** Create a new branch.
+
+```
+git checkout -b pr1
+```
+
+**8.** Search the "issue-finder" folder, add the modified HTML and CSS files and remove the previous ones.
```

Comment:
Perhaps we can simplify this to:

"Make the necessary changes to satisfy the requirements of the issue you are working on."